Salesforce_Advanced/Work_Dot_Com_Command_Center/WellnessForm.xml

```diff
@@ -0,0 +1,672 @@
+	<fields>
+		<field id="Employee__c"/>
+		<field id="Employee__r.Name"/>
+		<field id="StartDate__c"/>
+		<field id="Total_days__c"/>
+		<field id="Request_Type__c"/>
+	</fields>
+	<conditions>
+		<condition type="modelmerge" value="" field="Employee__c" fieldtargetobjects="Employee" operator="=" model="Employee" enclosevalueinquotes="true" mergefield="Id" novaluebehavior="deactivate"/>
+		<condition type="fieldvalue" value="1" enclosevalueinquotes="false" field="Total_days__c"/>
+		<condition type="fieldvalue" value="COVID19 Special" enclosevalueinquotes="true" field="Request_Type__c"/>
+		<condition type="fieldvalue" value="TOMORROW" enclosevalueinquotes="false" field="EndDate__c"/>
+		<condition type="fieldvalue" value="TODAY" enclosevalueinquotes="false" field="StartDate__c"/>
+	</conditions>
+	<actions/>
+</model>
+</models>
+	<components>
+		
+		<skuid__wrapper uniqueid="sk-ZtW-2206" styleSettingsVariant="2c157577-b1fd-4339-9850-4e33fefe1d9f">
+			<components>
+				<skuid__grid uniqueid="sk-aB1-5341" flexDirection="row" justifyContent="flex-start" alignItems="flex-start" columnGutter="4" rowGutter="4">
+					<divisions>
+						<division minWidth="240px" ratio="0" alignSelf="auto" maxWidth="240px">
+							<components>
+								<skuid__wrapper uniqueid="sk-bUG-5330" styleSettingsVariant="5d85cba1-676c-4e58-81fa-cd74d201bb5a">
+									<components>
+										
+									</components>
+									<background/>
+								<styles>
+<spacing/>
+</styles>
+</skuid__wrapper>
+							</components>
+						</division>
+						<division alignSelf="auto" ratio="3">
+							<components>
+								<skuid__header uniqueid="sk-Znz-4320" title="How are you doing? " model="" separator="false" styleSettingsVariant="29804294-53c1-4d4d-8040-c71bb104baa5">
+									<skuid__buttonSet>
+										<groups/>
+									</skuid__buttonSet>
+									<styles>
+										<spacing/>
+									</styles>
+								</skuid__header>
+							</components>
+						</division>
+					</divisions>
+					<styles>
+						<spacing bottom="3"/>
+					</styles>
+				</skuid__grid>
+				<skuid__grid uniqueid="sk-bUN-6055" flexDirection="row" justifyContent="flex-start" alignItems="flex-start" columnGutter="4" rowGutter="4">
+					<divisions>
+						<division minWidth="240px" ratio="0" alignSelf="auto" maxWidth="240px">
+							<components>
+								<skuid__wrapper uniqueid="sk-bUN-6056" styleSettingsVariant="5d85cba1-676c-4e58-81fa-cd74d201bb5a">
+									<components>
+										
+									</components>
+									<background/>
+									<styles>
+										<spacing/>
+									</styles>
+								</skuid__wrapper>
+							</components>
+						</division>
+						<division alignSelf="auto" ratio="3" minWidth="400px" maxWidth="1208px">
+							<components>
+								<skuid__header title="Please answer these questions every day" uniqueid="sk-29Qc-14607" titleTag="h2"/>
+								<skuid__text contents="&lt;p&gt;We care deeply about you&lt;/p&gt;&#10;" uniqueid="sk-bUN-6061">
```

Comment:
Hahaha I think this text is puts this into "Overly Attached Survey" territory.  I like the styled version at the bottom, so since we have that, can we cut this?

```suggestion
```